wordle-web/pages/WordEditor.vue

```diff
@@ -0,0 +1,319 @@
+<template>
+  <v-container fill-height justify-center>
+    <v-card>
+      <v-card-title class="display-3 justify-center">
+        Word Editor
+      </v-card-title>
+      <v-alert v-if="newWordSet" width="80%" :type="newWordResultType">
+          {{ newWordResultText }}
+        </v-alert>
+        <v-alert v-if="wordToDeleteSet" width="80%" :type="wordToDeleteResultType">
+          {{ wordToDeleteResultText }}
+        </v-alert>
+      <v-form v-if="authorizedToAddDelete" method="post" @submit.prevent="addWord">
+              <v-container>
+                <v-row>
+                  <v-col 
+                    cols="12" md="6"
+                  >
+                    <v-text-field
+                      v-model="newWord"
+                      label="New Word"
+                      required
+                    ></v-text-field>
+                  </v-col>
+                <v-col cols="12" md="6">
+                  <v-btn type="submit">Add Word</v-btn>
+                </v-col>
+              </v-row>
+              </v-container>
+            </v-form>
+      <v-row>
+        <v-col cols="12" sm="1"></v-col>
+        <v-col cols="12" sm="1">
+          <v-card-actions @click="firstPage">
+            <v-icon>mdi-page-first</v-icon>
+          </v-card-actions>
+        </v-col>
+        <v-col cols="12" sm="1">
+          <v-card-actions @click="decrementPage">
+            <v-icon>mdi-arrow-left-drop-circle</v-icon>
+          </v-card-actions>
+        </v-col>
+        <v-col cols="12" sm="5"></v-col>
+        <v-col cols="12" sm="1">
+          <v-card-actions @click="incrementPage">
+            <v-icon>mdi-arrow-right-drop-circle</v-icon>
+          </v-card-actions>
+        </v-col>
+        <v-col cols="12" sm="1">
+          <v-card-actions @click="lastPage">
```

Comment:
Last/first page buttons are really cool features, remember to set the page number variable to the end as well so the other buttons will work correctly after clicking last (first seems to already do it).